c10/core/TensorImpl.cpp

```diff
@@ -15,6 +15,16 @@ C10_DEFINE_int64(
     "The maximum memory in bytes to keep on shrink, if the difference between "
     "tensor sizes is bigger than this then tensor will be reset.");
 
+const std::string err_msg_tensor_metadata_change_not_allowed =
```

Comment:
nit: might be slightly better to make this a `const char * const` -- don't need the string include and no ABI issues.